ProcessMaker/Http/Middleware/GenerateMenus.php

```diff
@@ -153,7 +153,7 @@ public function handle(Request $request, Closure $next)
 
         Menu::make('sidebar_about', function ($menu) {
             $submenu = $menu->add(__('About'));
-            $submenu->add(__('Profile'), [
+            $submenu->add(__('Profile Settings'), [
```

Comment:
Recommend we change this to "Edit Profile" to better pair with "View Profile" as well as the breadcrumb navigation.